include/rcutils/strerror.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2017 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RCUTILS__STRERROR_H_
+#define RCUTILS__STRERROR_H_
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#include "rcutils/visibility_control.h"
+
+RCUTILS_PUBLIC
+void
+rcutils_safe_strerror(char * buffer, size_t buffer_length);
```

Comment:
@clalancette missing docblock

Reply:
Oops, yeah, good call.  Added in 9200c9fe916bb89d3e2721bd9d340cd83da598f2

Reply:
How about removing `safe` from the name of the function?

Reply:
I took the name from https://github.com/ros2/rclcpp/blob/2d9c6ea3a77103b91f7984269d163037961a4c04/rclcpp/src/rclcpp/signal_handler.cpp#L164 .  But I don't have a strong opinion, so we can just switch it to `rcutils_strerror` if you prefer.